src/fmu/tools/fipmapper/fipmapper.py

```diff
@@ -0,0 +1,320 @@
+"""The FipMapper class, mapping region/zones in RMS to FIPxxx in Eclipse.
+
+Assumptions:
+
+ * A FIPNUM always maps to an arbitrary-length list of regions
+ * A FIPNUM always maps to an arbitrary-length list of zones
+ * A region is assumed to present for all zones, but not relevant in this class
+ * A zone is assumed to be present for all regions, but not relevant in this class
+
+"""
+from pathlib import Path
+import logging
+from typing import Union, Dict, List, Any, Optional
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+class FipMapper:
+    def __init__(
+        self,
+        yamlfile: Union[str, Path] = None,
+        mapdata: Dict[str, str] = None,
+        skipstring: Union[List[str], str] = None,
+    ):
+        """FipMapper is a utility class for being able to map between
```

Comment:
Google docstrings: First line shall be a one liner summary.

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Fixed.